component/manage-service/src/main/java/com/wso2telco/dep/manageservice/resource/util/ApplicationConfiguration.java

```diff
@@ -34,4 +48,26 @@ public static Properties readAppConfig() {
         return properties;
 
     }
+
+    public static String getMsisdnValidationServiceUrl() {
+		String serviceEndpoint = "";
+		String carbonHome = System.getProperty("carbon.home");
+		String apiManagerFile = carbonHome + "/repository/conf/api-manager.xml";
```

Comment:
Let's have serviceEndpoint as a class level attribute,so that we can avoid reading api-manager.xml each time we call this method.